src/modules/roc_sndio/target_sox/roc_sndio/player.cpp

```diff
@@ -115,7 +128,7 @@ void Player::loop_() {
     SOX_SAMPLE_LOCALS;
 
     while (!stop_) {
-        pipeline::IReceiver::Status status = input_.read(frame);
+        pipeline::IReceiver::Status status = input_->read(frame);
```

Comment:
We should panic if run() was called and input_ is null. A panic is better than a segfault.

Reply:
Fixed